src/pages/Login/Login.tsx

```diff
@@ -2,6 +2,8 @@ import { FC, useEffect } from 'react'
 import { useNavigate, Link } from 'react-router-dom'
 import { loginUser } from '@/features/auth/authSlice'
 import { useAppDispatch, useAppSelector } from '@/app/hooks'
+import AuthButton from '../AuthButton/AuthButton'
```

Comment:
If this is a component, please put it into `components` folder

Reply:
Yes, it is a component. I will move it to ```components``` folder later